src/components/include/DefaultImportModalDialog.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import _ from 'underscore';
+import {Made} from "@exabyte-io/made.js";
+import {ModalHeader, ModalBody} from 'react-bootstrap';
+
+
+import {Material} from "../../material";
+import {ModalDialog} from './ModalDialog';
+import BasisText from "../source_editor/BasisText";
+
+
+export const defaultXYZMaterialConfig = {
```

Comment:
We should move this to Made.js - somewhere close to where Xyz parsers are, perhaps.